docs/operator-docs/provider.md

```diff
@@ -0,0 +1,77 @@
+---
+id: provider
+title: Provider Configuration
+hide_title: true
+---
+
+# Provider Configuration
+
+In Cumulus a Provider represents a endpoint from which data is ingested.   For example, a http server serving data from a data provider, or an S3 bucket deployed within your organization with data staged for ingest into Cumulus.
+
+Please note:
+
+* While *connection* configuration is defined here, things that are more specific to a specific ingest setup (e.g. 'What target directory should we be pulling from' or 'How is duplicate handling configured?') are generally defined in a Rule or Collection, not the Provider.
+* Task-specific provider configurations that are expected to be configured on a *per-workflow* basis exist, and may be tweaked to allow for task-specific configuration (e.g. https timeout values for the `discover-granules` task).
+
+A Provider can be created via use of the [API](https://nasa.github.io/cumulus-api/#create-provider) or via a client application like the [Cumulus Dashboard](https://github.com/nasa/cumulus-dashboard).
+
+## Provider Configuration
+
+The Provider configuration is defined by a JSON object that takes different configuration keys depending on the provider type.    The following are definitions of a typical configuration values relevant for the various providers:
+
+### S3
+
+|Key  |Type |Required|Description|
+|:---:|:----|:------:|-----------|
+|id|string|Yes|Unique identifier for the provider|
+|globalConnectionLimit|integer|No|Integer specifying the connection limit for the provider. This is the maximum number of connections Cumulus compatible ingest lambdas are expected to make to a provider.  Defaults to unlimited |
```

Comment:
Is this relevant from S3?

Reply:
I believe so, yes.     There's nothing in `ingest/src/lock.proceed` that cares about the provider type, nor in sync-granule (which is the only task to use that setup). 

That raises the point that `Cumulus compatible ingest lambdas` is a bit.... mushy as currently that's only SyncGranules.      DiscoverGranules/PDR doesn't care because it's periodicity is controlled by the Rule setting and/or the lambda timeout.     🤔     Perhaps we should write a ticket or at least discuss if discovery should respect this limit as it's not terribly global at the moment.

Reply:
As long as it is relevant for S3, the other concerns are secondary for now